### ISSUE TITLE
Adding a language

### ISSUE THREAD

**Issue by MyBlood** [gmetais/YellowLabTools#398]
Friends, can you tell me how to add additional localization correctly?

Thanks.

**Comment by gmetais** [comment]
Hello @MyBlood,
Unfortunately, YellowLabTools was not designed to be easily translated. It would require a long development.
What is your use case?

**Comment by MyBlood** [comment]
> What is your use case?

If I understand you correctly, I'm using YellowLab through Docker



> Unfortunately, YellowLabTools was not designed to be easily translated. It would require a long development.

There is no separate file responsible for translating the interface?

> Hello @MyBlood,

Greetings!

**Comment by gmetais** [comment]
No, there's no separate file and it would take a very long time to make it possible.
By use case, I mean is it for your personal use or for sharing the tool in any way, and which language would you like to add?

**Comment by MyBlood** [comment]
> No, there's no separate file and it would take a very long time to make it possible. By use case, I mean is it for your personal use or for sharing the tool in any way, and which language would you like to add?

Greetings! I would like to translate it into Russian. Yes, for personal use, it may be useful for others in the public domain)
